### PR TITLE
Added chalk for pretty output colors

### LINKED ISSUE #18: Use pretty colors in the output?
https://github.com/chalk/chalk can do it for you, though if you want just an angry red & bold color it's easy to do inline.

I'd be willing to send a PR too!

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Fixed #18.

Might want to be verified by someone on a *nix machine that it isn't atrocious there... but it's through chalk so I doubt it.

image

### CODE REVIEW

**Comment by parshap** [comment]
The `grey` colors don't work well for me sshed into Ubuntu via Putty using the Solarized color scheme:

image

Maybe just stick to white/red/bold.

File: bin.js [hunk 1]
```diff
@@ -43,8 +44,8 @@ function printVersions(result, print) {
     if (!isSatisfied) {
       var raw = info.wanted.raw;
       var range = info.wanted.range;
-      console.log("Error: Wanted " + name + " version " + raw + " (" + range +")");
-      console.log(PROGRAMS[name].getInstallInstructions(raw));
+      console.error(chalk.red.bold("Error: Wanted " + name + " version " + raw + " (" + range +")"));
+      console.log(chalk.red.bold(PROGRAMS[name].getInstallInstructions(raw)));
```
Comment (parshap) [inline bin.js]: Maybe just the error message should be red and the installation instructions could be normal color? Or are you thinking more red to grab attention?

**Comment by parshap** [comment]
Looks good now!

image

File: bin.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply bin.js]: I was thinking more red to grab attention. Otherwise it felt like "info" text along with the rest of the display.

Comment (parshap) [reply bin.js]: Sounds good to me! Thanks for adding this.

**Comment by parshap** [comment]
Published as v3.1.0.

**Comment by paulmelnikow** [comment]
This looks really nice!

**Pull request merged**